adam-core/src/main/java/org/bdgenomics/adam/io/FastqRecordReader.java

```diff
@@ -60,6 +60,24 @@
      * application.  We'll see if someone complains in other applications.
      */
 
+    /** Default maximum read length, <code>10,000</code> bp. */
+    public static final int DEFAULT_MAX_READ_LENGTH = 10000;
+
+    /** Maximum read length property name. */
+    public static final String MAX_READ_LENGTH_PROPERTY = "org.bdgenomics.adam.io.FastqRecordReader.MAX_READ_LENGTH";
```

Comment:
Is there a way to set this in the cli or just the shell?

Reply:
Hmm, good point.

It is a property in the Hadoop configuration, so I suppose
```
adam-submit \
  --conf org.bdgenomics.adam.io.FastqRecordReader.MAX_READ_LENGTH=42 \
  --
```
might work